tests/WorkOS/UserManagementTest.php

```diff
@@ -105,7 +105,10 @@ public function testAuthorizationURLInvalidInputs()
     public static function authorizationUrlTestDataProvider()
     {
         return [
+            [null, null, Resource\ConnectionType::AppleOAuth, null],
+            [null, null, Resource\ConnectionType::GitHubOAuth, null],
             [null, null, Resource\ConnectionType::GoogleOAuth, null],
+            [null, null, Resource\ConnectionType::MicrosoftOAuth, null],
```

Comment:
Seeing this test fail with `AppleOAuth` helped me find the `$supportedProviders` array above. Doesn't hurt to run it for all our social logins.